### ISSUE TITLE
Downloading boxart fails with IPv6 enabled

### ISSUE THREAD

**Issue by RaymiiOrg** [TeamShinkansen/Hakchi2-CE#171]
On Windows 10, hakchi v3.5.2, Europe SNES.

Box art, both right click Im lucky, or the Google button and selecting manually, fail for me when IPv6 is enabled (ncpa.cpl, network connection properties, check the IPv6 box). 

My ISP provides native IPv6 and hakchi tries do do IPv6 google, but it seems to timeout or just fail:

![2018-11-29 17_43_27-connection history](https://user-images.githubusercontent.com/1927494/49236930-76310680-f3fd-11e8-86d2-950ebe9b8625.png)

When unchecking the ipv6 box, downloading boxart works.

**Comment by rhester72** [comment]
Does going to https://ipv6.google.com/ in your browser work?

**Comment by RaymiiOrg** [comment]
Yes. All other IPv6 connectivity works, my ISP provides a native connection (no tunnels like HE.net)

**Comment by ghost** [comment]
Is it DS-Lite or real IPv6 your providers supplies ?
Try to set another DNS Server for your network adapter, try to use WLAN instead of cable...
It looks like there is a lot of encrypted traffic there the selected one is HTTP (non encrypted) have you tried to open that particular address in your browser while IPv4 disabled  ?.

you can proto your ssl traffic trough eg. fiddler and watch the traffic through wireshark.
you could setup a system proxy (net.framework) to pass the traffic through this and test if this will work.

there could be made more tests ;)
